components/admin_console/billing/plan_details/plan_details.tsx

```diff
@@ -137,48 +145,49 @@ export const planDetailsTopElements = (userCount: number, isPaidTier: boolean, i
                 />
             </div>
         );
+
+        productName = (
+            <FormattedMessage
+                id='admin.billing.subscription.planDetails.productName.mmCloud'
+                defaultMessage='Mattermost Cloud'
+            />
+        );
     }
-    const productName = (
-        <FormattedMessage
-            id='admin.billing.subscription.planDetails.productName.cloudProfessional'
-            defaultMessage='Cloud Professional'
-        />
-    );
 
-    // switch (typeSubscription) {
-    // case 'FREE_TRIAL':
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.cloudProfessional'
-    //             defaultMessage='Cloud Professional'
-    //         />
-    //     );
-    //     break;
-    // case 'CLOUD_ENTERPRISE':
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.cloudEnterprise'
-    //             defaultMessage='Cloud Enterprise'
-    //         />
-    //     );
-    //     break;
-    // case 'CLOUD_STARTER':
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.cloudStarter'
-    //             defaultMessage='Cloud Starter'
-    //         />
-    //     );
-    //     break;
-    // default:
-    //     productName = (
-    //         <FormattedMessage
-    //             id='admin.billing.subscription.planDetails.productName.mmCloud'
-    //             defaultMessage='Mattermost Cloud'
-    //         />
-    //     );
-    //     break;
-    // }
+    switch (subscriptionPlan) {
+    case 'CLOUD_PROFESSIONAL':
+        productName = (
+            <FormattedMessage
+                id='admin.billing.subscription.planDetails.productName.cloudProfessional'
+                defaultMessage='Cloud Professional'
+            />
+        );
+        break;
+    case 'CLOUD_ENTERPRISE':
+        productName = (
+            <FormattedMessage
+                id='admin.billing.subscription.planDetails.productName.cloudEnterprise'
+                defaultMessage='Cloud Enterprise'
+            />
+        );
+        break;
+    case 'CLOUD_STARTER':
+        productName = (
+            <FormattedMessage
+                id='admin.billing.subscription.planDetails.productName.cloudStarter'
+                defaultMessage='Cloud Starter'
+            />
+        );
+        break;
+    default:
+        productName = (
+            <FormattedMessage
+                id='admin.billing.subscription.planDetails.productName.cloudProfessional'
+                defaultMessage='Cloud Professional'
+            />
+        );
+        break;
+    }
```

Comment:
This switch statement can also just compare against the SKU instead of these customer _ cases. And they can use the constants you made earlier :)